src/views/mentorship/ui/mentors-after-register/ui/mentors-after-register.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen } from '@testing-library/react';
+import { beforeEach, describe, expect, it } from 'vitest';
+
+import { MentorsAfterRegister } from './mentors-after-register';
+
+const latinSymbolsRegExp = /^[?!,.a-zA-Z0-9\s]+$/;
+const cyrillicSymbolsRegExp = /^[?!,.а-яА-ЯёЁ0-9\s]+$/;
```

Comment:
Not sure how robust this approach is, as it will fail if text such as `'будет отправлено вам на электронную почту перед началом менторинга'` is replaced with `'будет отправлено вам на email перед началом менторинга'`

And wouldn't we want to test that it's rendering the values that we're passing, not some random text? Because for some components we have mocked data:
![image](https://github.com/user-attachments/assets/03f9e009-fb0a-4094-a0bc-6a5fbc8fc06b)

Reply:
Tests failed anyway if we change the data. If we would like to change smth in `data` file, we can to remove this cases or change them. About mocks - I guess it makes sense when we're testing a component that receives dynamic data (`props`), in my case we have static data (text) and I don't see the point in replacing them with mocks.